libs/dal/src/repositories/job/job.schema.ts

```diff
@@ -95,6 +95,10 @@ const jobSchema = new Schema(
     providerId: {
       type: Schema.Types.String,
     },
+    _actorId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Subscriber',
+    },
```

Comment:
We are already defining a property to bring the Subscriber reference in the line 40. Do we need them to store a different subscriberId for the job?

Reply:
Yes, we need it, cause the subscriber and actor will be two different users
it's like User A likes User B's posts,
so User A = actor, User B = subscriber

Reply:
Understood!